src/main.functional.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai';
+import { app } from './main';
+
+describe('functional tests', () => {
+  /*eslint-disable */
+  //there are no types-supertest, so we use require.
+  //this throws lint error.
+  const request = require('supertest');
+  /*eslint-enable*/
+
+  describe('health endpoint', () => {
```

Comment:
error callback test? can spy on sentry call

Reply:
i am just looking to setup a skeleton for functional test, i will let product squad to continue with adding other test for their use cases.